merlin/systems/dag/ops/softmax_sampling.py

```diff
@@ -89,7 +89,7 @@ def compute_output_schema(
         self, input_schema: Schema, col_selector: ColumnSelector, prev_output_schema: Schema = None
     ) -> Schema:
         """Describe the operator's outputs"""
-        return Schema([ColumnSchema("ordered_ids", dtype=np.int32, is_list=True, is_ragged=True)])
+        return Schema([ColumnSchema("ordered_ids", dtype=np.int32, is_list=True, is_ragged=False)])
```

Comment:
The value of `is_ragged` didn't used to matter here, since we didn't have a clear differentiation between how ragged and non-ragged lists were handled. It does now though, and this is a fixed-length list since we're picking the top-k elements.